src/AllLayers.js

```diff
@@ -37,15 +39,14 @@ L.LayerGroup.environmentalLayers = L.LayerGroup.extend(
 
     initialize: function(param) {
       param = param || {};
```

Comment:
Could you try adding `L.Util.setOptions(this, param);` above this line?

Reply:
And then remove lines 40 to 48 and lines 61 to 69 and try how it works?

Reply:
Oh yes that's a good point. I'll add your suggestions!